pandas/core/generic.py

```diff
@@ -3080,6 +3086,10 @@ def to_csv(
                supported for compression modes 'gzip' and 'bz2'
                as well as 'zip'.
 
+            .. versionchanged:: 1.2.0
+
+                Compression is supported for non-binary file objects.
```

Comment:
should have been "Compression is supported for **binary** file objects."! @jreback Should I create a one-line PR for this?

Reply:
I opened a PR: #35615